src/main/java/gov/nih/nci/evs/api/fhir/R4/FhirUtilityR4.java

```diff
@@ -638,6 +679,43 @@ public static BundleLinkComponent getNextLink(
     return new BundleLinkComponent().setUrl(nextUri).setRelation("next");
   }
 
+  /**
+   * Get the previous link component
+   *
+   * @param uri the uri
+   * @param offset the offset
+   * @param offsetInt the offset int
+   * @param count the count
+   * @param countInt the count int
+   * @return the previous link component
+   */
+  public static BundleLinkComponent getPrevLink(
+      final String uri,
+      final NumberParam offset,
+      final int offsetInt,
+      final NumberParam count,
+      final int countInt) {
+    final int prevOffset = offsetInt - countInt;
+    String prevUri = uri;
+    // append ? to url if missing
+    if (!uri.contains("?")) {
+      prevUri = prevUri + "?";
+    }
+    // replace offset if it exists
+    if (offset != null) {
+      prevUri = prevUri.replaceFirst("_offset=\\d+", "_offset=" + prevOffset);
+    } else {
+      prevUri += (prevUri.endsWith("?") ? "" : "&") + "_offset=" + prevOffset;
+    }
+    // replace count if it exists
```

Comment:
I can see why we replace the offset. I am not sure what the reason is to replace the count when it does not change? When it does not exist, will we get a countInt passed in?

Reply:
I'm not sure honestly. This method was modeled after the getNextLink() and I didn't want to deviate too far from that implementation. But I'm also not sure why we are replacing the count and if that is the correct approach for looking at previous links. @bcarlsenca Do you have insight/feedback on this?